### DISCUSSION TITLE
Concurrent User Capacity for Web App Service Plan SKU S1 Standard

### DISCUSSION THREAD

**Discussion by ravikhunt** [microsoft/PubSec-Info-Assistant#807] [General]
I am currently using an Azure Web App Service Plan with the following default settings:

File Location: /infra/variables.tf
SKU Settings:
appServiceSkuSize: S1
appServiceSkuTier: Standard

Based on your experience and these default application settings, can you estimate the number of concurrent users this configuration can support? 

Also can you please tell us the number of user support if we increase the SKU Size , it's important for our implementation

**Comment by dayland** [comment]
This was answered in https://github.com/microsoft/PubSec-Info-Assistant/issues/808
